lib/fakedata/src/logs.rs

```diff
@@ -0,0 +1,220 @@
+use chrono::{prelude::Local, SecondsFormat};
```

Comment:
Is the original file deleted?

Reply:
This is required for the `timestamp_*` functions

Reply:
Nope! My bad. It is now.